### DISCUSSION TITLE
Changing timezone in Frigate

### DISCUSSION THREAD

**Discussion by oprso** [blakeblackshear/frigate#1841] [Ask A Question]
Hi, how can i change time zone in frigate which i couldn't find it in the document.
The green timestamp in the pic was generated from my camera stream and the white one was the frigate one which is 12 hr off.
I am currently in +12 time zone.

Thanks

![Capture](https://user-images.githubusercontent.com/89329222/134754699-5dc437ca-3e17-4811-b761-cf5d00692aff.JPG)

**Comment by blakeblackshear** [comment]
If you are running in docker, it will pull from the host with the volume mappings in your docker command. Also see here: https://github.com/blakeblackshear/frigate/issues/1818#issuecomment-926900477
